pkg/ipcache/metadata.go

```diff
@@ -572,5 +568,5 @@ type identityUpdater interface {
 }
 
 type policyTriggerer interface {
```

Comment:
nit: this interface now could be renamed to policyUpdater/policyMapUpdater, just for readability and consistency with identityUpdater

Reply:
agree, I've been working on some additional refactoring on top of this change and this is one of the things I am already thinking about. Currently thinking of "datapathHelper.UpdatePolicyMaps" since this piece is logically about how the ipcache propagates policy information into the datapath (+Envoy).